File: Flutter/flutter_socket/lib/models/candidate.dart
class Candidate {
  String id;
  String name;
  String candidate;
  int votes;

  Candidate({this.id, this.name, this.candidate, this.votes});

  factory Candidate.fromMap(Map<String, dynamic> candidate) => Candidate(
      id: candidate['id'],
      name: candidate['name'],
      candidate: candidate['candidate'],
      votes: candidate['votes']);
}

File: models/candidate.js
const { v4: uuid_v4 } = require('uuid');



class Candidate {
    constructor(id, name = "no-name", postulacion, votes) {
        this.id = id; //Identificador unico
        this.name = name;
        this.postulacion = postulacion;
        this.votes = votes;
    }
}

module.exports = Candidate;